docs/source/conf.py

```diff
@@ -128,6 +128,20 @@
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
+# Ensure all our internal links work
+nitpicky = True
+nitpick_ignore = [
+    ('py:obj', 'ipywidgets.HBox'),
+    ('py:obj', 'k3d.plot.Plot'),
```

Comment:
Is this really needed, as there is an intersphinx mapping for k3d?

Reply:
I didn't find a documentation page for the `Plot` class, so that's probably why also Sphinx couldn't find it.

Reply:
Ah, ok.